src/main/java/org/frc5687/steamworks/protobot/OI.java

```diff
@@ -79,6 +79,10 @@ public double getRightSpeed(){
         return transformStickToSpeed(Gamepad.Axes.RIGHT_Y);
     }
 
+    public boolean getLeftTrigger(){
```

Comment:
Rename this to better indicate what the Boolean return value represents (i.e. IsLeftTriggerPressed)